src/Catalog/SortingCollector.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace NuGet.Services.Metadata.Catalog
 {
-    public abstract class SortingCollector : CommitCollector
+    public abstract class SortingCollector<T> : CommitCollector
```

Comment:
Interesting...they should have written it so that `IEquatable<T>` should require reimplementation of `GetHashCode`...